docs/contributing.rst

```diff
@@ -89,10 +89,12 @@ Releasing the theme
 
 To release a new version of the theme, core team will take the following steps:
 
-#. Bump the version in ``sphinx_rtd_theme/__init__.py``, ``setup.py`` and
-   ``package.json``.  We follow `semver <http://semver.org/>`_ and `PEP440`_
+#. Bump the version by running ``bump2version [major|minor|patch|dev]``.
+   We follow `semver <http://semver.org/>`_ and `PEP440`_
    (with regards to alpha release and development versions). The version
    increment should reflect these releases and any potentially breaking changes.
+#. If the new version is not a dev version, run ``bump2version --allow-dirty release`` to
```

Comment:
Is `release` a string, or the release version number? I assume it's a string `release`.

Reply:
Yes it is the exact string ``release``